aws-samples/aws-secure-environment-accelerator
# add subnetcidr pools

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

This fixes the Security Group rules when referencing a different subnets. The code's lookup functionality was always returning empty because it had no data populated to query.

To verify:

    In a workload account, inspect the Mgmt security group. With the default config, it is expected to see ingress rules with descriptions of 'Central VPC Traffic Inbound from...'
    Apply fix and run state machine, expected Security Group Ingress rules exist.
